electricitymap/contrib/parsers/tests/test_VN.py

```diff
@@ -0,0 +1,105 @@
+
+import pytest
+from datetime import datetime, timedelta
+from unittest.mock import ANY
+
+
+from electricitymap.contrib.parsers.lib.exceptions import ParserException
+
+PATCH_PATH = "electricitymap.contrib.parsers.VN" 
+
+
+class ZoneKey:
+    def __init__(self, zone): self.zone = zone
+    def __eq__(self, other): 
+        if isinstance(other, ZoneKey): return self.zone == other.zone
+        return self.zone == other
+    def __hash__(self): return hash(self.zone)
+    def __str__(self): return self.zone
```

Comment:
Why is this a class?

ZoneKey is just a type alias of str to help with type flows.